packages/sdk/src/types/AdaptedWallet.ts

```diff
@@ -34,4 +34,11 @@ export type AdaptedWallet = {
   address: () => Promise<string>
   switchChain: (chainId: number) => Promise<void>
   transport?: CustomTransport | HttpTransport
+  // evm wallets that support EIP-5792's atomic batch capability
+  // https://www.eip5792.xyz/capabilities/atomicBatch
+  supportsAtomicBatch?: (chainId: number) => Promise<boolean>
```

Comment:
We could do this, or we could just check if handleBatchTransactionStep is defined. Wdyt is best?

Reply:
The issue with that is `handleBatchTransactionStep` is always defined for the viem adapter, which is the default adapter that is used. We need to use the `wallet_getCapabilities` rpc to see if the wallet the user is using supports atomic batch

Reply:
oh I see, yeah that makes sense. I thought all evm wallets handled it.